WARNING Module not found: 'vue-class-component'

I am not using Vue Class Components so when I run the build it works but it displays a warning message:

```
WARNING in ./node_modules/vuex-pathify/dist/vuex-pathify.esm.js
Module not found: Error: Can't resolve 'vue-class-component' in '.../node_modules/vuex-pathify/dist'
@ ./node_modules/vuex-pathify/dist/vuex-pathify.esm.js 1072:20-50
```

The line in question is:
```javascript
try {
  createDecorator = require('vue-class-component').createDecorator;
} catch(e) {
  // Decorators are not available, so there is no need for decorators.
  createDecorator = function () { throw new Error("'vue-class-component' is required for decorators. Use 'npm -i vue-class-component'") };
}
```

Is there any way to use pathify without Vue Class Components and silence the warning message?

--- a/package.json
+++ b/package.json
@@ -32,7 +32,8 @@
     "url": "https://github.com/davestewart/vuex-pathify/issues"
   },
   "dependencies": {
-    "lodash.clonedeep": "^4.5.0"
+    "lodash.clonedeep": "^4.5.0",
+    "vue-class-component": "^7.0.2"
   },
   "devDependencies": {
     "install": "^0.12.2",
@@ -45,7 +46,6 @@
     "rollup-plugin-uglify": "^3.0.0",
     "typescript": "^3.2.2",
     "vue": "^2.5.20",
-    "vue-class-component": "^6.3.2",
     "vuex": "^3.0.1"
   },
   "peerDependencies": {

--- a/src/helpers/decorators.js
+++ b/src/helpers/decorators.js
@@ -15,17 +15,7 @@
 
 import { get, sync, call } from "./component"
 
-let createDecorator
-
-// Developers not useing `vue-class-component` do not need decorators.
-// Require is put in try-catch block to avoid installation of `vue-class-component` for those who do not need.
-// Developers using class components already have to include `vue-class-component`. So this require will use it.
-try {
-  createDecorator = require('vue-class-component').createDecorator
-} catch(e) {
-  // Decorators are not available, so there is no need for decorators.
-  createDecorator = () => { throw new Error("'vue-class-component' is required for decorators. Use 'npm -i vue-class-component'") }
-}
+let createDecorator = require('vue-class-component').createDecorator;
 
 /**
  * Decortaor for `get` component helper.

